core/templates/dev/head/pages/library/Library.js

```diff
@@ -65,6 +65,16 @@ oppia.controller('Library', [
     $scope.pageMode = LIBRARY_PATHS_TO_MODES[currentPath];
     $scope.LIBRARY_PAGE_MODES = LIBRARY_PAGE_MODES;
 
+    // Set page title
```

Comment:
This comment is not needed I think that it is fairly obvious what we are doing here.

Reply:
Okay, great.